resources/validation_rules/PHM_made_up_settings_2.json

```diff
@@ -0,0 +1,188 @@
+[
```

Comment:
🎨 Worth calling this file something like `PHM_validation_rules` or something? The other PHM file is a prototype, so should be fine?

Reply:
I might change this on the next ticket I'm on